### PR TITLE
Feature/new circle admin page

### LINKED ISSUE #962: 1.5.6 - Admin Updates Move circle admin settings from modals to new page
- Take the Settings modal and move it to a new page: /circles/[circleid]/admin/settings

### PR DESCRIPTION
**Pull request by sshmm**
## Motivation and Context

resolves #962

## Description
What is done :
1- The main design of CircleAdminPage
2- Header link
3-RadioGroupForm updated with stitched and Radix and useController from https://react-hook-form.com/
4- General Section of the CircleAdminPage

To be done:
1- Complete the Epoch Section "Preview Part"
2- Complete the logic of the epoch section.
 
## Test and Deployment Plan

to be added after finishing the Epoch section

## Screenshots (if appropriate):

image

image

### CODE REVIEW

**Comment by levity** [comment]
Hey @sshmm, thanks for the work on this so far. Since there are a lot of pieces to be changed in the new admin UI, it might be good to first create a small PR that does nothing but move the existing design from a modal to a standalone page.

File: src/routes/paths.ts [hunk 1]
```diff
@@ -37,7 +37,8 @@ const withSearchParams = (
 
 export const paths = {
   // circle-specific
-  adminCircles: (circleId: number) => `/circles/${circleId}/admin`,
+  adminCircles: (circleId: number) => `/circles/${circleId}/members`,
```
Comment (levity) [inline src/routes/paths.ts:40]: can you change the path name? `adminCircles` doesn't make any sense anymore

**Comment by sshmm** [comment]
> Hey @sshmm, thanks for the work on this so far. Since there are a lot of pieces to be changed in the new admin UI, it might be good to first create a small PR that does nothing but move the existing design from a modal to a standalone page. As you can see in `main`, I pushed some changes so that circle-specific routes now have the circle ID in the URL.

 I almost finished the epoch timing logic. unless you don't want these changes to be introduced this time

**Comment by crabsinger** [comment]
perfect 💯

**Pull request closed without merging**